src/browser/modules/App/App.tsx

```diff
@@ -149,6 +150,11 @@ export function App(props: any) {
     props.bus && props.bus.send(initAction.type, initAction)
   }, [props.bus])
 
+  useEffect(() => {
+    const pageLoadAction = trackPageLoad()
+    props.bus && props.bus.send(pageLoadAction.type, pageLoadAction)
+  }, [props.bus, props.telemetrySettings.allowUserStats])
```

Comment:
I think it shouldn't be a problem, but I still worry a little about this effect running extra times and giving us bad stats. I think we could be certain to avoid that if we run dispatch in `dbMetaEpics` instead, in the section where the server configuration finishes. There's a comment: `// side-effects` where it'd fit well

Reply:
thanks! will take a look